### PR TITLE
Add type field customer model aschoice

### PR DESCRIPTION
**Pull request by paodanchacon**

### CODE REVIEW

File: inventory/models.py [hunk 1]
```diff
@@ -21,8 +28,13 @@ def __str__(self):
 
 
 class Location(models.Model):
+    class Type(models.TextChoices):
+        PHYSICAL = "PH", _("Physical")
+        PARTNER = "PA", _("Partner")
+        VIRTUAL = "VI", _("Virtual")
```
Comment (tabris2015) [inline inventory/models.py:34]: Can you elaborate what are the differences between these types of customers?

**Review by tabris2015 — COMMENTED** [review]
Left a comment

File: inventory/models.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 
 class Customer(models.Model):
+    class Type(models.TextChoices):
+        RETAIL = "RE", _("Retail")
+        WHOLESALE = "WH", _("Wholesale")
+
```
Comment (tabris2015) [inline inventory/models.py:9]: Can we add a new `partner` customer type in order to identify customers that we don't necessarily sell to or buy from?

**Comment by paodanchacon** [comment]
Outdated pull request

**Pull request closed without merging**